packages/block-editor/src/components/caption/index.native.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+const Caption = ( { accessible, accessibilityLabel, onBlur, onChange, onFocus, isParentSelected, isSelected, text } ) => {
+	// We still want to render the caption so that the soft keyboard is not forced to close on Android
+	const shouldCaptionDisplay = ! RichText.isEmpty( text ) > 0 || isParentSelected;
+
+	return (
+		<View style={ {	padding: 12, flex: 1, display: shouldCaptionDisplay ? 'flex' : 'none' } }
+			accessible={ accessible }
+			accessibilityLabel={ accessibilityLabel }
+			accessibilityRole={ 'button' }
+		>
+			<RichText
+				rootTagsToEliminate={ [ 'p' ] }
+				placeholder={ __( 'Write caption…' ) }
+				value={ text }
+				onChange={ onChange }
+				unstableOnFocus={ onFocus }
```

Comment:
Because `Caption`'s `onFocus` prop is being passed on to `RichText` an _unstable_ `RichText` prop (`unstableOnFocus`), perhaps I should also name this prop for caption `unstableOnFocus` instead of just `onFocus`?

Reply:
I think is fine? If `unstableOnFocus` changes, I guess we can modify the behavior internally to keep the caption.onFocus stable.